src/utils/cpu.py

```diff
@@ -49,11 +44,14 @@ def get_cpu_tdp(cpu_name: str) -> float:
     :return: CPU Thermal Design Power (TDP) in watts
     :rtype: float
     """
+    # check that the input is a string otherwise raise a ValueError
+    if not isinstance(cpu_name, str):
```

Comment:
Also here, cpu_name is not chosen by the user, it is the output of cpu.get_cpu_info(), same below